examples/controls_example/public/react_controls/data_controls/types.ts

```diff
@@ -26,12 +26,13 @@ export type DataControlApi = DefaultControlApi &
 
 export interface DataControlFactory<
   State extends DefaultDataControlState = DefaultDataControlState,
-  Api extends DataControlApi = DataControlApi
+  Api extends DataControlApi = DataControlApi,
```

Comment:
Are these type changes still needed?

Reply:
Removed in bb5cfb6989f8e5738c2766dd7ee0fabab9801a47 :+1: